tests/testthat/test-eda.R

```diff
@@ -56,3 +56,41 @@ test_describe_num_var <- function() {
     regexp = "Only numeric columns expected, please check the input.")
   })
 }
+
+
+test_describe_cat_var <- function() {
```

Comment:
I think we need to add documentation for the test function as well. It is required in milestone 2.

Reply:
added documentation